app/src/main/java/org/mozilla/tv/firefox/channels/content/NewsChannels.kt

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.channels.content
+
+import org.mozilla.tv.firefox.R
+import org.mozilla.tv.firefox.channels.ChannelTile
+import org.mozilla.tv.firefox.channels.TileSource
+
+fun ChannelContent.getNewsChannels(): List<ChannelTile> = listOf(
```

Comment:
To be fair, our current `assets/json` strategy is also part of the source, and would also require a code push to change.  The only difference here is what format the data is kept in.  This also still has a very clear seam where we could easily convert it to JSON if we needed to make that change, it seems like overengineering to me to do that now when we don't know whether or not we'll need it.

Reply:
Storing raw "untouched" files under `assets` is a standard to application development. An example can be found [here](https://github.com/googlesamples/android-sunflower/tree/master/app/src/main/assets). To simplify the implementation (if you think it causes too much overhead), we can use [ `AssetFileDescriptor`](https://developer.android.com/reference/android/content/res/AssetFileDescriptor). Many RESTful apps use raw asset data as a backup to API polling failure.

Reply:
It also creates inconsistency with bundled pinned tiles implementation. We are bundling a set of similar data format, we should look to centralize them.

Reply:
Our bundled/pinned tiles are already inconsistent with each other, and will need to be refactored anyway.  Storing data as JSON is definitely a common practice, but I'm not seeing the benefit in this case.  We can't update our JSON without a code push, we have no easy way to translate the JSON (unlike this solution, where we can use localized strings like normal), JSON provides very little in the way of type safety or validation, it is substantially less performant (a trivial difference, but still), etc.

If we ever do have a use case where we need this to be JSON, we can convert it.  But I don't think it would be any less work to convert it right now.  Why not keep the benefits we get from this approach until we have a compelling reason to do otherwise?

Reply:
Two few thoughts here:
- Using the data class directly is certainly easier code-wise
- In the future, we (might) need to add more files, created by not necessarily technical people, and I think the idea of passing .kt files around def seems kind of gross 😅 
- However, the data class is compile-time format checked, which is definitely better than JSON.
- As for inconsistency between bundled/pinned tiles, I guess they'd need to be updated to match. We definitely shouldn't have two formats for this.

I guess both sides both have good reasons, and I like the separation of data files from code, but in this case, I think data files is totally fine.

Reply:
>I'm not seeing the benefit in this case

In engineering, scalable solutions tend to be more optimized because you require less rewiring whenever requirements change. Do you mean that you don't see the benefits right now? or in the future, say for example, if we want to have multiple sources to the data set?

Reply:
Tbh, I don't see real future benefits.  If we eventually need to support pulling in new channels at runtime, we will still need to inflate them into a list of domain objects.  So we sit the IO and parsing in front of this point, and otherwise use the same code path.  It's an easy change, and it's not any easier to make it right now than it would be to do it when we need it.

Reply:
OK, can you change the pinned tile bundle stuff to source code data files and deprecate the `BundleTileStore`?

Reply:
I'll try.  Custom/bundled tile inflation is pretty weird and tangled right now, and it might be out of scope for this story.  But if I can get it to work in a reasonable time frame, I will.

Reply:
> I'll try. Custom/bundled tile inflation is pretty weird and tangled right now, and it might be out of scope for this story. But if I can get it to work in a reasonable time frame, I will.

@Baron-Severin okay then let's create a follow-up story

Reply:
Opening now.  I'll link it here.